packages/core/src/Stores/notification-store.js

```diff
@@ -614,7 +595,19 @@ export default class NotificationStore extends BaseStore {
                           },
                           text: localize('Give feedback'),
                       }
-                    : notification_redirect_action,
+                    : {
+                          onClick: () => {
+                              this.p2p_order_props.redirectToOrderDetails(order_id);
+                              this.p2p_order_props.navigateToOrderDetails(order_id);
```

Comment:
what is the difference between `redirectToOrderDetails` and `navigateToOrderDetails`? can we keep just one?

Reply:
```
redirectToOrderDetails(order_id) {
        const { order_store } = this.root_store;
        this.redirectTo('orders');
        this.setOrderTableType(order_list.INACTIVE);
        order_store.setOrderId(order_id);
    }
```

```
const navigateToOrderDetails = id => {
        history.push({ pathname: routes.p2p_orders, search: `?order=${id}` });
    };
```

Reply:
We need to have the navigate to order details passed into the p2p_order_props so we can route using history.push. because we can't call it in the stores itself where redirectToOrderDetails is. But looking at this maybe we can instead get rid of the redirectToOrderDetails and simply call the other functions in navigateToOrderDetails:

```
     const navigateToOrderDetails = id => {
        history.push({ pathname: routes.p2p_orders, search: `?order=${id}` });
        general_store.setActiveIndex(1);
        general_store.setOrderTableType(order_list.INACTIVE);
        order_store.setOrderId(id);
    };
```

Reply:
@niloofar-deriv @ameerul-deriv yes let's keep one function if possible